eclipse/test/junit/saros/editor/internal/ContributionAnnotationManagerTest.java

```diff
@@ -300,4 +357,24 @@ private int getAnnotationCount(IAnnotationModel model) {
 
     return positions;
   }
+
+  private User createAliceTestUser() {
```

Comment:
As the returned object is always the same, these methods could also be replaced by a static variable for each user. Or are the user objects modified by the tests?